src/beta_inc.jl

```diff
@@ -968,12 +967,7 @@ function _beta_inc_inv(a::Float64, b::Float64, p::Float64, q::Float64=1-p)
     sq = 1.0
     prev = 1.0
 
-    if x < 0.0001
-        x = 0.0001
-    end
-    if x > .9999
-        x = .9999
-    end
+    x = clamp(x, fpu, prevfloat(1.0))
```

Comment:
This change, or rather the lower bound `fpu`, break examples such as https://github.com/JuliaStats/StatsFuns.jl/issues/145. Unfortunately, the alternative `1e-200` proposed in https://github.com/JuliaMath/SpecialFunctions.jl/pull/396/ breaks that example as well.